helmfile.d/values/networkpolicies/service/grafana.yaml.gotmpl

```diff
@@ -27,7 +27,7 @@ policies:
               podSelectorLabels:
                 app.kubernetes.io/component: query-frontend
           ports:
-            - tcp: 10902
+            - tcp: 9090
```

Comment:
So, is grafana and grafana-label-enforcer updated to use these ports? Or does it use port names?

Reply:
I did not find that this was needed as they are targeting the service, e.g. https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile.d/values/grafana/grafana-label-enforcer.yaml.gotmpl#L3
Unless I'm missing something? :thinking:

Reply:
(Since the containerPort is what has changed)

Reply:
Ah, strange :sweat_smile: